autoload/asyncomplete.vim

```diff
@@ -211,6 +211,12 @@ function! s:should_skip() abort
     endif
 endfunction
 
+function! s:get_min_chars(source_name) abort
+    let l:source = s:sources[a:source_name]
+    let l:min_chars = has_key(l:source, 'min_chars') ? l:source['min_chars'] : g:asyncomplete_min_chars
```

Comment:
I think you should be able to do this directly

```vim
function! s:get_min_chars(source_name) abort
    return get(l:source, 'min_chars', g:asyncomplete_min_chars)
endif
```

Reply:
Updated, thanks!